lib/repl.js

```diff
@@ -1126,7 +1126,10 @@ function isRecoverableError(e, self) {
       self._inTemplateLiteral = true;
       return true;
     }
-    return /^(Unexpected end of input|Unexpected token)/.test(message);
+
+    return message.startsWith('Unexpected end of input') ||
+      message.startsWith('Unexpected token') ||
+      message.startsWith('missing ) after argument list');
   }
```

Comment:
Perhaps a better approach would be defining a const as:
```
const remsg = new RegExp(
  '^(Unexpected end of input|' +
  'Unexpected token|' +
  'missing \\) after argument list)'
);
```
and using `return remsg.test(message);` ?